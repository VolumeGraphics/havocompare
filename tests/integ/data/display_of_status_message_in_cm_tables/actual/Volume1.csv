Name,Status
Cylinder 1,"Isosurface not found for iteration > 0, but previous result could be taken"
Cylinder 2,OK
0.0, 0.1