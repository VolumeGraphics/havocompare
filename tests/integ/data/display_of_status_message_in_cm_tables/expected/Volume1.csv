Name;Status
Cylinder 1;"Isosurface not found for iteration > 0, but previous results could be taken"
Cylinder 2;OK
0,0; 0,1